BUILD

```diff
@@ -47,3 +47,21 @@ toolchain(
     toolchain = ":py_runtime_pair",
     toolchain_type = "@rules_python//python:toolchain_type",
 )
+
+py_binary(
```

Comment:
Now that we have this working in bazel, let's move the `watch.py` into the `docs` dir.